### PR TITLE
feat(nx-dev): when feedback button is pressed hide buttons

### PR DESCRIPTION
**Pull request by mandarini**
## Current Behavior

A user clicks the feedback button and there's no indication of whether the feedback went through.

## Expected Behavior

There should be some sort of indication that the feedback went through.

Step 1 - Click button:
Screenshot 2023-08-02 at 1 25 24 PM

Step 2 - Buttons disappear - "Thank you for your feedback" is displayed
Screenshot 2023-08-02 at 1 25 28 PM

## Notes

I think this can be considered a temporary solution while we're still working with this existing UI. When we switch to a more designed, chat-like interface, maybe this can be integrated in there?

### CODE REVIEW

**Comment by MaxKless** [comment]
Works well but the toast is on the complete opposite of the screen from the buttons. Even though I know the toast exists I missed it on a large screen :P Maybe it could be centered?

**Comment by FrozenPandaz** [comment]
Instead of a toast, I'd replace the feedback buttons after they are pressed. We don't want them to be pressed twice so we should just remove them.

**Comment by mandarini** [comment]
@FrozenPandaz I replaced the toast with your suggestion which makes more sense! :)

File: nx-dev/feature-ai/src/lib/feature-ai.tsx [hunk 1]
```diff
@@ -73,43 +86,43 @@ export function FeatureAi(): JSX.Element {
...
+              </Button>
+            </div>
+          )}
+          {feedbackSent && (
+            <p>
+              <span role="img" aria-label="check">
+                ✅
+              </span>{' '}
+              Thank you for your feedback!
+            </p>
```
Comment (FrozenPandaz) [inline nx-dev/feature-ai/src/lib/feature-ai.tsx:124]: I think we can still make this prettier but it's low priority and can be done in a follow up.

**Review by FrozenPandaz — APPROVED** [review]

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**